lib/mockbuilder.py

```diff
@@ -69,6 +69,31 @@ def build(self, package):
         LOG.info("%s: Starting build process" % package.name)
         self._build_srpm(package)
         self._install_external_dependencies(package)
+        self._build_rpm(package)
+
+        msg = "%s: Success! RPMs built!" % (package.name)
+        self._copy_rpms(self.build_dir, package.build_cache_dir)
+        LOG.info(msg)
```

Comment:
Could you move this log to inside `_build_rpm`? It does not make much sense to surround `_copy_rpms` with these lines.